internal/dag/cache.go

```diff
@@ -36,6 +36,7 @@ import (
 // A KubernetesCache holds Kubernetes objects and associated configuration and produces
 // DAG values.
 type KubernetesCache struct {
+
```

Comment:
Is this extra space needed?

Reply:
Nope, just reads better with it personally. I'll take out.